apps/prairielearn/src/pages/partials/submitRateFooter.ejs

```diff
@@ -1,21 +0,0 @@
-<% if (showGradeButton && typeof assessment_question !== "undefined" && assessment_question && assessment_question.grade_rate_minutes) { %>
-<div class="row">
-  <div class="col d-flex justify-content-between">
-  <span class="d-flex">
-  <% if (disableGradeButton) { %>
-    <small class="font-italic ml-2 mt-1 submission-suspended-msg">Grading possible in <span id='submission-suspended-display'></span> <div id='submission-suspended-progress' class='border border-info'></div> </small>
-    <%- encoded_data({
```

Comment:
This call and the one removed by #9974 are the last references to `encoded_data` in locals. Once #9974 is merged into master and then into this PR it can be removed from locals.

Reply:
Removed in a4aa464.